### PR TITLE
feat(login form): New login form layout

### PR DESCRIPTION
**Pull request by madole**
- Added a new log in form style
- Added subtle pattern background
- Added a temporary logo
- Added a temporary logo with text

### CODE REVIEW

**Comment by chrislaughlin** [comment]
Build failing, please only merge if the build passes in the pull request process.

**Comment by madole** [comment]
Looks like the CSS lint is failing the build. It didn't fail when I did the PR but we're moving to SASS at some point which I'll spend time converting the CSS to this week. 
Did you check out the login form?

**Comment by martinmicunda** [comment]
The pull request build passed okay that's why i have merged the code (for pull request we are not publishing any artifacts). The reason why normal build failed is because of this command in ci_build.sh file 

```
git rm -rf .
fatal: pathspec '.' did not match any files
```

Basically when we are running normal build it blank gh-pages branch and then it publish new artifacts there. I think for failed build it never published any artifacts so when build is running again with command `git rm -rf .` there are no any files in gh-pages branch so this command failed... I think it' s because we also running build on our fork branches....

Basically the test should run against our fork branches and not against cam repo.. I need to figure out that.

**Comment by chrislaughlin** [comment]
can you add a if in the script that will check if there are files to remove first

**Comment by martinmicunda** [comment]
yeah i can do that but basically that should never happend...

**Comment by martinmicunda** [comment]
@madole that logo and login form looking good andrew :) Just one quick suggestion... The login page should be default one and the links in top right corner should be hidden on login page.. once user login the links should be visible and there should be logout link and not login...

**Comment by chrislaughlin** [comment]
We should follow the wire frames, maybe revisit them and have a planning
session on them.

On 8 June 2014 15:35, Martin Micunda notifications@github.com wrote:

> @madole https://github.com/madole that logo and login form looking good
> andrew :) Just one quick suggestion... The login page should be default one
> and the links in top right corner should be hidden on login page.. once
> user login the links should be visible and there should be logout link and
> not login...
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cam-technologies/time-booker/pull/25#issuecomment-45438232
> .

## 

Regards

Chris Laughlin

**Comment by martinmicunda** [comment]
yeah agree with that

**Comment by madole** [comment]
Yeah, the wireframe and the current update can be merged together with a bit more work. I just had a few free hours today so go playing with it. Work in progress! :)
I'm cool with a planning session and I agree with @martinmicunda on the log in page being the first page and the hidden links.
